server/push_test.go

```diff
@@ -1533,7 +1533,9 @@ func TestPush_ReuseIntmidWithDiffExtmid(t *testing.T) {
 
 func TestPush_MultipartReturnMultiSession(t *testing.T) {
 	assert := assert.New(t)
-
+	//need real video data for fast verification
+	transcodeddata, err := ioutil.ReadFile("../core/test.ts")
```

Comment:
Does it test the fast verification? If yes, then which check `pHash` or `CompareVideoByBuffer`? I think we should have somewhere tests for both.

Reply:
Will add unit test  for `CompareVideoByBuffer`  separately.